ztp/policygenerator-kustomize-plugin/testPolicyGenTemplate/common-ranGen.yaml

```diff
@@ -7,6 +7,11 @@ spec:
   bindingRules:
     common: "true"
   sourceFiles:
+    # Create inform policy to validate configuration CRs that will be applied to all clusters
+    - fileName: validatorCRs/informDuValidator.yaml
+      complianceType: musthave
+      remediationAction: inform
+      policyName: "du-validator-policy"
```

Comment:
I don't think we should be copying all the published content into the test data directory. This adds confusion as to where the active code really is. If the added YAML presents a new type of input then creating generic test data here makes sense. In this case it looks like policy with remediationType of "inform" is new and it makes sense to add in tests to cover that.

Reply:
Updated.